src/styles/_color.scss

```diff
@@ -0,0 +1,10 @@
+$primary-text-color: #6c757d;
+$primary: #5d9143;

```

Comment:
we have $primary, $primary-text-color & $primary-txt-color. Is it not confusing? We can name the variable as the color it represents. So it is easier to identify